<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevSetting_Amp_2_0_Mic_Line</class>
 <widget class="QWidget" name="DevSetting_Amp_2_0_Mic_Line">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMP Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:checked{
	background-color: rgb(0, 170, 0);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>A-In1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="micgain_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <item>
    <property name="text">
     <string>0dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30dB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>36dB</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="micphantom_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>60</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Phantom</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="gain_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-60</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QPushButton" name="xover_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Low Cut</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dyn_1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Dyn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eq_1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Eq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phase_1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Mixer">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Mixer</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="outgain_1">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-60</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>260</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N-Out1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>290</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>N-Out2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dyn_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Dyn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eq_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Eq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phase_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Phase</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="xover_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Low Cut</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="outgain_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-60</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>A-In2</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="gain_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-60</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Mic_open">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mic_open</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AIn1_Channel_Mute">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AIn2_Channel_Mute">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NOut1_Channel_Mute">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="NOut2_Channel_Mute">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>290</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>mute</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_In_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_In_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>111</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>N1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>N2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_Out_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_Out_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
